Integrations/integration-CheckpointFirewall

```diff
@@ -0,0 +1,707 @@
+    - contextPath: CheckpointFWRule.Type
+      description: Type of the object.
+    - contextPath: CheckpointFWRule.Action
+      description: Accept, Drop, Apply Layer, Ask, Info. How much details are returned
+        depends on the details-level field of the request. This table shows the level
+        of detail shown when details-level is set to standard.
+    - contextPath: CheckpointFWRule.ActionSetting
+      description: Action settings.
+    - contextPath: CheckpointFWRule.CustomFields
+      description: Custom fields.
+    - contextPath: CheckpointFWRule.Data
+      description: How much details are returned depends on the details-level field
+        of the request. This table shows the level of detail shown when details-level
+        is set to standard.
+    - contextPath: CheckpointFWRule.Data.Name
+      description: Object name. Should be unique in domain.
+    - contextPath: CheckpointFWRule.UID
+      description: Object unique identifier.
+    - contextPath: CheckpointFWRule.Type
+      description: Type of the object.
+    - contextPath: CheckpointFWRule.Data.Domain
+      description: Information about the domain the object belongs to.
+    - contextPath: CheckpointFWRule.DataDirection
+      description: On which direction the file types processing is applied.
+    - contextPath: CheckpointFWRule.DataNegate
+      description: True if negate is set for data.
+    - contextPath: CheckpointFWRule.Destination
+      description: Collection of Network objects identified by the name or UID. How
+        much details are returned depends on the details-level field of the request.
+        This table shows the level of detail shown when details-level is set to standard.
+    - contextPath: CheckpointFWRule.DestinationNegate
+      description: True if negate is set for destination.
+    - contextPath: CheckpointFWRule.Domain
+      description: Information about the domain the object belongs to.
+    - contextPath: CheckpointFWRule.Domain.Name
+      description: Object name. Should be unique in domain.
+    - contextPath: CheckpointFWRule.Domain.UID
+      description: Object unique identifier.
+    - contextPath: CheckpointFWRule.Domain.Type
+      description: Domain type.
+    - contextPath: CheckpointFWRule.Enabled
+      description: Enable/Disable the rule.
+    - contextPath: CheckpointFWRule.Hits
+      description: Hits count object.
+    - contextPath: CheckpointFWRule.Hits.FirstDate
+      description: First of hits.
+    - contextPath: CheckpointFWRule.Hits.LastDate
+      description: Last date of hits.
+    - contextPath: CheckpointFWRule.Hits.Level
+      description: Level of hits.
+    - contextPath: CheckpointFWRule.Hits.Percentage
+      description: Percentage of hits
+    - contextPath: CheckpointFWRule.Hits.Value
+      description: Value of hits.
+    description: Block one or more IP addresses using Checkpoint Firewall.
+  - name: checkpoint
+    arguments: []
```

Comment:
Since this !checkpoint should work with any given arg, and required args override the others (bug) I had to remove 'command' from the arguments. The 'checkpoint' command makes sure that 'args.command' is defined.